lbrynet/core/utils.py

```diff
@@ -37,6 +37,14 @@ def today():
     return datetime.datetime.today()
 
 
+def timedelta(**kwargs):
+    return datetime.timedelta(**kwargs)
+
+
+def datetime_obj(*args, **kwargs):
+    return datetime.datetime(*args, **kwargs)
```

Comment:
The reason for the other ones is that "now" needs to be repeatable / mockable for tests.  I don't think we don't need to mock out timedelta or datetime.datetime